src/main/java/application/rest/v1/ActionsEndpoint.java

```diff
@@ -733,27 +752,27 @@ private void setSecurityContextAndServiceAccountName(ApiClient client, V1PodSpec
         spec.setServiceAccountName(config.getkAppNavServiceAccountName()); 
     }
     
-    private Map<String,String> createJobAnnotations(String text) {
+    private Map<String,String> createJobAnnotations(String text, String user) {
         if (text != null && !text.isEmpty()) {
```

Comment:
This should be used to determine whether the "text" annotation should be set. Setting the user id is independent of that. Need to massage the logic here a bit.

Reply:
Good catch, @mrglavas.